packages/api/src/providers/fitbit.ts

```diff
@@ -332,9 +335,13 @@ export class Fitbit extends Provider implements OAuth2 {
       "Accept-Language": "en_GB", // For higher precision in weight readings, we are retrieving data in stones and converting it to kg
     };
 
+    const fitbitToken = connectedUser.providerMap?.fitbit?.token;
+    const scopes = fitbitToken ? JSON.parse(fitbitToken).scope : [];
+    const containsProfile = scopes.includes("profile");
```

Comment:
`scopes` is `any`, isn't it? So calling `.includes()` is not safe - it might be `undefined` and not be an array.

Reply:
Its actually a string but i will check the type of before hand

Reply:
sounds good. If its a string then maybe the default value should be `undefined` and not `[]`